components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -346,18 +339,13 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
     }, []);
   }
 
-  detailParamsChange(queryParams) {
-    const sgId = queryParams.get('sgId');
-    if (sgId) {
-      const servicesFilters: GroupServicesFilters = {
-        service_group_id: sgId,
-        page: parseInt(queryParams.get('sgPage'), 10) || 1,
-        pageSize: parseInt(queryParams.get('sgPageSize'), 10) || 25,
-        health: queryParams.get('sgStatus') || 'total',
+  refreshServicesSidebar() {
+    if (this.selectedServiceGroupId) {
+      this.store.dispatch(new UpdateSelectedSG({
+        service_group_id: this.selectedServiceGroupId,
```

Comment:
Unlike the previous example I commented on, thi argument to `UpdateSelectedSG` is strongly typed already because it is inlined, so no changes needed.